### PR TITLE
Added TLS 1.2 Note

### PR DESCRIPTION
**Pull request by pupara**
Added TLS 1.2 Note

### CODE REVIEW

File: skype/skype-ps/skype/Move-CsUser.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ When moving a user to or from the Microsoft 365 cloud (TeamsOnly):
...
 > - Prior to retirement of Skype for Business Online, organizations that still need to move users from on-premises to Skype for Business Online can do so uing a two-step process: First move the user from on-premises to TeamsOnly using `-Move-CsUser`, and after the move change the mode of the user to something other than TeamsOnly using `Grant-CsTeamsUpgradePolicy`. However, the ability to change a cloud user's mode to something other than TeamsOnly will soon be removed as part of Skype for Business Online retirement.
+> - Moving users from On prem to Teams requires TLS 1.2. TLS 1.0 and TLS 1.1 have been deprecated. Please visit [Disabling TLS 1.0 and 1.1 for Microsoft 365](https://docs.microsoft.com/microsoft-365/compliance/tls-1.0-and-1.1-deprecation-for-office-365?view=o365-worldwide) and [Preparing for TLS 1.2 in Office 365 and Office 365 GCC](https://docs.microsoft.com/microsoft-365/compliance/prepare-tls-1.2-in-office-365?view=o365-worldwide) for details.
```
Comment (get-itips) [inline skype/skype-ps/skype/Move-CsUser.md:54]:
```suggestion
> - Moving users from On-Premises to Teams requires TLS 1.2. TLS 1.0 and TLS 1.1 have been deprecated. Please visit [Disabling TLS 1.0 and 1.1 for Microsoft 365](/microsoft-365/compliance/tls-1.0-and-1.1-deprecation-for-office-365?view=o365-worldwide) and [Preparing for TLS 1.2 in Office 365 and Office 365 GCC](/microsoft-365/compliance/prepare-tls-1.2-in-office-365?view=o365-worldwide) for details. 
```

**Comment by yogkumgit** [comment]
@pupara

**Comment by yogkumgit** [comment]
@pupara Gentle reminder, please implement the feedback. Thanks

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**